content/about/sponserende_leden.erb

```diff
@@ -0,0 +1,63 @@
+---
+narrow_page: true
+---
+
+<%= render '/partials/_about_sub_navbar.*', selected: 'sponsoring_members' %>
+<h1 class='title is-1 has-text-centered'>Sponserende Leden</h1>
+
+<div id='sponsoring_members' class="content">
+  <% current_sponsoring_members.each do |member| %>
+      <div id="<%= member[:nickname] %>" class="box sponsoring_member">
+        <article class="columns is-mobile">
+          <div class="column is-gapless is-narrow logo-wrapper">
+            <% if member[:photo] %>
+              <div class="sponserend_image">
+                <img src="<%=member[:photo]%>">
+              </div>
+            <% else %>
+              <div class="sponserend_image" style="background-color: #ff7f00">
+                <div class="logo-initial has-text-centered">
+                  <%= member[:name].chars.first %>
+                </div>
+              </div>
+            <% end %>
+            <p class="name"><%= member[:name] %></p>
+            <p class="username">(<%= member[:nickname] %>)</p>
+          </div>
+          <div class="text_wrapper">
+          <div class="column text_data">
+            <div>
+              <i class="quote">
+                "<%= member[:quote] %>"
+              </i>
+            </div>
+            <div class="description_text" style="margin-top: 0.5em">
+              <%= member[:description] %>
+            </div>
+          </div>
+          <div class="columns links">
+              <% if member[:github] %>
+                <a style="padding: 5px" class="social-icon" href="https://github.com/<%= member[:github] %>" target="_blank">
+                  <span class="icon">
+                    <%= fa :github, {:size => "2x"} %>
+                  </span>
+                </a>
+              <% end %>
+              <% if member[:twitter] %>
+                <a style="padding: 5px" class="social-icon" href="https://twitter.com/<%= member[:twitter] %>" target="_blank">
+                  <span class="icon">
+                    <%= fa :twitter, {:size => "2x"} %>
+                  </span>
+                </a>
+              <% end %>
+              <% if member[:blog] %>
```

Comment:
Misschien is het een idee om de icoontjes en website verticaal uit te lijnen.
![Aantekening 2019-11-18 175323](https://user-images.githubusercontent.com/1756811/69072705-70786a80-0a2c-11ea-9f61-5178942c4044.png)